src/Components/ArticleList.jsx

```diff
@@ -19,7 +19,6 @@ const ArticleList = () => {
 
   return (
     <main className="Articles">
-      <Navbar/>
       <ul className="Article-list">
         {ArticleList.map((article) => {
           return (
```

Comment:
you need to include a Link to the single article route, at the moment there's no way for the user to access it unless they manually type in the url